### ISSUE TITLE
$:/status/IsLoggedIn ?

### ISSUE THREAD

**Issue by joshuafontany** [OokTech/TW5-Bob#148]
I am running a set of wikis with a local Bob node.js wsserver. Does Bob ever set the text of `$:/status/IsLoggedIn` to "yes" when logging in with the basic username:password found in the primary wiki's settings.json file?

I am trying to use the `tiddlywiki/consent-banner` plugin's feature of assuming consent for "logged in users", but Bob doesn't seem to do so.

Best,
Joshua Fontany
---

I am running (check any that apply, put an x inside the [ ] to check a box, like this: [x]):

- [x] Windows
- [ ] OSX
- [ ] Linux
- [ ] Other

and using

- [x] The nodejs version
- [ ] The single file executable

Before posting I read [issue guidelines](./IssueGuidelines.md) and:

- [x] I am using the newest version
- [x] The answer to my question isn't listed in the documentation or this isn't
  a question
- [x] This is not a duplicate issue
- [x] I have not done anything that required me to set acceptance to
  `I Will Not Get Tech Support For This`

**Comment by inmysocks** [comment]
No, at the moment Bob doesn't have any interaction with the basic login method for the core server.
The is logged in tiddler is for the secure server that I made and which is in desperate need of updates.

There are a lot of things I want to do to make bob work with the core, I will add putting compatibility with the core login options to the list.

**Comment by inmysocks** [comment]
I am trying to make Bob fit more with the core so that more can me moved from Bob to the core, so $:/status/IsLoggedIn now gets set, but with Bob there isn't a login mechanism so it only has an effect on the secure server. I hope to release a working version of the secure server very soon.
